README.md

```diff
@@ -24,42 +24,64 @@
 
 ### Day 1
 
-10:00-11:00 [Intro to ES6 Classes](https://github.com/oliverjam/es6-class-intro)  
-11:00-11:30 [Modern frontend & how React fits in](https://hackmd.io/p/SJauYz6EM#) (talk/discussion)  
-11:30-13:00 [React API](https://github.com/oliverjam/intro-react-workshop) (workshop)  
-_13:00-14:00 Lunch_  
-14:00-15:00 [React API](https://github.com/oliverjam/intro-react-workshop) (workshop continued)  
-15:00-16:00 [Build a stopwatch](https://github.com/oliverjam/intro-react-workshop/blob/master/workshop-top-notch-stopwatch) (mini project)  
+08:30-09:00 Introduction to React Week and learning outcomes.
+
+09:00-10:00 [Intro to ES6 Classes](https://github.com/oliverjam/es6-class-intro)
+
+10:05-10:30 [Modern frontend & how React fits in](https://hackmd.io/p/SJauYz6EM#) (talk/discussion)
+
+10:30-12:00 [React API](https://github.com/oliverjam/intro-react-workshop) (workshop)
+
+12:00-13:00 -- LUNCH --
+
+13:00-15:00 [React API](https://github.com/oliverjam/intro-react-workshop) (workshop continued)
```

Comment:
seem's like we are giving this 3.5 hours vs. 2.5 hours in the original?  I would say give the stop watch more time instead?

Reply:
I think that's a typo in the original as well, presumably mini project should go on till 18:00 in london